dashboard/config/scripts_json/allthemigratedthings.script_json

```diff
@@ -626,5 +632,14 @@
   ],
   "lessons_opportunity_standards": [
 
+  ],
```

Comment:
Note, I believe these were only added because this lesson hasn't been saved for a few years and the rubrics were recently added.